src/core/admin-core.js

```diff
@@ -0,0 +1,31 @@
+import _ from 'lodash';
+const {knex} = require('../util/database').connect();
+import {GCS_CONFIG} from '../util/gcs';
+//import CONST from '../constants';
+const logger = require('../util/logger')(__filename);
+//import moment from 'moment-timezone';
+
+
+
+//Admin deleteFeedItem same as in feed-core.js but doesn't need opts.client.uuid 
```

Comment:
Tälläset muistiinpanot imo kans pois, ellei oo jotain mikä ihan oikeasti tarvii selittämistä

Reply:
Eli heitän kaikki tosta 10rivistä alaspäin olevat kommentit? 
Vai jätänkö ohjeet kuinka testata?